lib/delayed_job_web/application/views/layout.erb

```diff
@@ -2,8 +2,8 @@
 <html>
   <head>
     <title>Delayed Job Web</title>
-    <link rel="stylesheet" type="text/css" href="stylesheets/reset.css" />
-    <link rel="stylesheet" type="text/css" href="stylesheets/style.css" />
+    <link rel="stylesheet" type="text/css" href="<%= u('stylesheets/reset.css') %>" />
+    <link rel="stylesheet" type="text/css" href="<%= u('stylesheets/style.css') %>" />
```

Comment:
this was necessary (and seemingly an oversight on upstream/master). if we don't do this, then anytime you render a path ending in `/` that isn't `/delayed_job/` these files will not be found.